runtime/container.go

```diff
@@ -204,9 +205,23 @@ func (c *container) readSpec() (*specs.Spec, error) {
 	return &spec, nil
 }
 
+func (c *container) GetRuntimeRoot() string {
+	runtimeRoot := "/run/runc"
+
+	// Check that the root wasn't changed
```

Comment:
Actually, yes.

Just need this one for the call to `libcontainer.New`, will update